public/config/configuration.json

```diff
@@ -15,10 +15,12 @@
                 "falseColor": "#ff6666"
             },
             "data":{
-                "topic": "telemetry",
-                "topicAttribute": "charger_has_power",
+                "scale":1,
```

Comment:
README_configuration.md says this configuration attributes is:
data.scale : (joystick) floating point values array [x, y]

Reply:
that's true for joystick, not indicator/state

Reply:
the difference is market with (indicator) vs (joystick) for the property,

Reply:
The only entry in README_configuration.md is:

data.scale : (joystick) floating point values array [x, y]

Reply:
updated scale to array in configuration and connected to joystick widget